pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -126,6 +129,7 @@ func (pc *PodCache) deleteIP(ip string) {
 	pod := pc.podsByIP[ip]
 	delete(pc.podsByIP, ip)
 	delete(pc.IPByPods, pod)
+	pc.recentDeletedPods[podKey{ip, pod}] = struct{}{}
```

Comment:
Don't see this being read, is it needed?

Reply:
Can this cache be used in determining endpoint address corresponds to a deleted pod?

Reply:
It is not needed now. I will remove it.